docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+networks:
+  db-data-net:
+    driver: "bridge"
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: my-sql:1.2
```

Comment:
The image name `my-sql:1.2` seems unconventional. Typically, official MySQL images are named `mysql`. Verify that this image exists or is correctly built locally.